src/modules/ping.js

```diff
@@ -1,8 +1,9 @@
 import { SlashCommandBuilder } from '@discordjs/builders';
 import { registerCommandModule } from '../utils/commands.js';
 
-const pingCommand = new SlashCommandBuilder()
-  .setDescription('replies with pong');
+const pingCommand = new SlashCommandBuilder().setDescription(
```

Comment:
Try not to include unnecessary formatting changes in your commits.

Reply:
Didn't notice this. Prettier did whatever it did, I'll try not to change in the future.